src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ProjectReference.xaml

```diff
@@ -112,6 +112,11 @@
                     Description="Version of reference.">
     </StringProperty>
 
+    <StringProperty Name="GeneratePathProperty" 
```

Comment:
What does this represent? This name is not clear to me.

Reply:
this generates msbuild property `Pkg<Package_Name>` which points to the package root path. This is what I've tried to explain in description. Let me know if you've better text.

`Generates msbuild property to the path of this reference`

Reply:
"Indicates whether to generate an MSBuild property with the location of the package's root directory. The generated property name is in the form of 'Pkg[PackageID]', where '[PackageID]' is the ID of the package with any periods '.' replaced with underscores '_'."